Mathlib/Algebra/Group/Subgroup/Map.lean

```diff
@@ -253,13 +253,19 @@ theorem map_inf_eq (H K : Subgroup G) (f : G →* N) (hf : Function.Injective f)
 theorem map_bot (f : G →* N) : (⊥ : Subgroup G).map f = ⊥ :=
   (gc_map_comap f).l_bot
 
-@[to_additive (attr := simp)]
+@[to_additive]
 theorem map_top_of_surjective (f : G →* N) (h : Function.Surjective f) : Subgroup.map f ⊤ = ⊤ := by
   rw [eq_top_iff]
   intro x _
   obtain ⟨y, hy⟩ := h x
   exact ⟨y, trivial, hy⟩
 
+@[to_additive (attr := simp)]
+lemma map_top_of_mulEquivClass {G N F : Type*} [Group G] [Group N]
+    [EquivLike F G N] [MulEquivClass F G N] (f : F) :
```

Comment:
`G` and `N` are already in scope here, so we don't need to redeclare them.

Not sure I like the `of_mulEquivClass` in the title, that seems quite unwieldy. Maybe something like this? It would match the `Submonoid` result.
```suggestion
lemma map_equiv_top {F : Type*} [EquivLike F G N] [MulEquivClass F G N] (f : F) :
```